content/realtime/inband-occupancy.textile

```diff
@@ -0,0 +1,124 @@
+---
+title: Inband channel occupancy events
+meta_description: "Inband occupancy events are occupancy events for a channel published to the channel itself."
+meta_keywords: "Ably, inband, occupancy, occupancy events, channel metadata."
+section: realtime
+index: 5
+jump_to:
+  Help with:
+    - Overview#overview
+    - Occupancy channel parameter#occupancy-channel-parameter
+    - Occupancy events#occupancy-events
+    - See also#see-also
+---
+
+h2(#overview). Overview
+
+The inband channel occupancy feature allows a client to subscribe to occupancy metadata events relating to a single channel, and for those events to be delivered to the client as messages on the channel itself. Occupancy metadata events received as messages on the channel are known as __inband events__.
+
+The @Occupancy@ type is defined in the "Channel Metadata documentation":/realtime/channel-metadata#occupancy.
+
+h2(#occupancy-channel-parameter). Occupancy channel parameter
+
+Inband channel occupancy events are enabled in a channel attachment by specifying the @occupancy@ parameter. This is described in more detail in the "Channel Parameters documentation":/realtime/channel-params. The value of the parameter specifies the occupancy fields of interest, and the supported parameter values are:
+
+* @metrics@: this enables events containing the full @Occupancy@ details in their @data@ payload. Events are sent when the count for any of the included categories changes, up to a maximum rate of one message per second.
```

Comment:
The exact delay is hard to quantify legibly since it's the sum of a couple different rollup / debouncing timers at different layers (one 8s, one 5s). But 1s is very optimistic.

Maybe sthg like "Events are sent when the count for any of the included categories changes. Updates that involve mode changes (e.g. at least one publisher existing where there were none before) are propagated immediately; ones that do not are debounced, for no more than 15 seconds."

(similarly next para)

Reply:
Fixed in both cases.